packages/build/src/log/messages/core_steps.js

```diff
@@ -17,6 +17,24 @@ const logBundleResultFunctions = ({ functions, headerMessage, logs, error }) =>
   logArray(logs, functionNames)
 }
 
+const MAX_AWS_ZIP_SIZE = 50000000
+
+const logSizeWarnings = ({ results, logs }) => {
+  const oversizedFunctions = results.filter(({ size }) => size > MAX_AWS_ZIP_SIZE)
+  oversizedFunctions.forEach(({ name, bundler }) => {
+    logErrorSubHeader(logs, `Function ${name} exceeds size limit of 50mb`)
+
+    if (bundler === 'zisi') {
+      logMessage(`You're using the default bundler, and could benefit from esbuild: https://ntl.fyi/try-esbuild`)
```

Comment:
@eduardoboucas while I'm working on adding a test for this, what do you think about the messaging?

Reply:
I'd like us to check with Integrations first, because I believe they've added a similar message to the Next.js plugin. We want to make sure we don't have conflicting/confusing messaging.

Reply:
(cc @ascorbic)

Reply:
I'll go ahead and merge, we're free to update the messaging in a follow-up PR.

Reply:
@netlify-team-account-1 Can we check with Integrations first? This is not an urgent fix, so I'd rather wait and get this right than introduce confusion for Next.js users.